src/lua/zencode_qp.lua

```diff
@@ -118,104 +157,97 @@ ZEN.add_schema(
 		       export = O.to_hex },
       kyber_ciphertext = { import = kyber_ciphertext_f,
 			   export = O.to_hex },
-      kyber_kem = { import = import_kem,
-		    export = export_kem }
+      kyber_kem = { import = kyber_import_kem,
+		    export = kyber_export_kem },
+
+      
+      ntrup_public_key = { import = ntrup_public_key_f,
```

Comment:
the "complex" schema defining import/export functions can be simplified, just one function is enough and will use the encoding set by `rule` or default base64. this can be done since there is no established standard for encoding ntru keys, so any encoding is good. other cases use import/export where a specific standard is needed (i.e in bitcoin, eth...)

can explain this in a meeting soon